Dockerfile

```diff
@@ -1,16 +1,24 @@
-FROM golang:1.9.6 AS build
+FROM golang:1.10-alpine AS build
 ADD . /src
 WORKDIR /src
-RUN go get -d -v -t
-RUN go test --cover ./... --run UnitTest
-RUN go build -v -o docker-flow-proxy
+RUN set -x \
+    && apk add --update --no-cache --no-progress git \
+    && go get -d -v -t \
+    && go test --cover ./... --run UnitTest \
+    && go build -v -o docker-flow-proxy
 
 
-FROM haproxy:1.8.8-alpine
-MAINTAINER 	Viktor Farcic <viktor@farcic.com>
+FROM haproxy:1.8-alpine
+LABEL org.opencontainers.image.title="Docker Flow Proxy" \
+    org.opencontainers.image.version="18.08.26" \
```

Comment:
The jenkins CI server generates the version depending on the date. Hard coding the version into the Dockerfile would most likely lead the mismatching of the tag and this version label.